google/cloud/firestore_v1/base_batch.py

```diff
@@ -146,9 +162,15 @@ def delete(
                 state of the document before applying changes.
         """
         write_pb = _helpers.pb_for_delete(reference._document_path, option)
+        self._document_references[reference._document_path] = reference
         self._add_write_pbs([write_pb])
 
-    def _prep_commit(self, retry, timeout):
```

Comment:
it seems like this is still used, is there a reason to remove it from the base?

Reply:
It's now moved to `BaseWriteBatch`, which is the base class of the pre-existing `[Async]WriteBatch` class. This negative delta is in `BaseBatch`, which handles logic shareable between `[Async]WriteBatch` and `BulkBatch`.